File: b0302hello/app/src/main/java/com/example/b0302hello/MainActivity.java
package com.example.b0302hello;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import java.util.ArrayList;
import java.util.Arrays;

public class MainActivity extends AppCompatActivity {

    private ListView listView;
    private String[] array;
    private ArrayAdapter arrayAdapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        listView = findViewById(R.id.listView);
        array = getResources().getStringArray(R.array.array1);
        arrayAdapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, new ArrayList(Arrays.asList(array)));
        listView.setAdapter(arrayAdapter);

    }
}

File: l2022string/app/src/main/java/com/example/l2022string/MainActivity.kt
package com.example.l2022string

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val textView1: TextView? = findViewById<TextView>(R.id.textView1)
        textView1?.setText("efgh");
    }
}